inc/ruleticket.class.php

```diff
@@ -768,6 +778,11 @@ function getActions() {
       $actions['solution_template']['table']                 = 'glpi_solutiontemplates';
       $actions['solution_template']['force_actions']         = ['assign'];
 
+      $actions['task_template']['name']          = _n('Task template', 'Task templates', 1);
+      $actions['task_template']['type']          = 'dropdown';
+      $actions['task_template']['table']         = TaskTemplate::getTable();
+      $actions['task_template']['force_actions'] = ['assign'];
```

Comment:
1. Align `=`with above actions.

2. i think, it could be better to set `append` action instead `assign` (for understanding  purpose) you need to check if the process is really different. 

3. Could it be possible to assign multiple time a task template to the same rule ? To have the feature complete, we should do it if it's possible.

Reply:
1. Done

2. Dev is already in production, if we change it it will break when the client update.
This dev was inspired by the `solution_template` rule action that also use `assign`.
IMO it would be better to have a separate "code improvement" PR that move then both to `append` and migrate the data if needed.

3. Does not seem possible with current rule system (you can only add each actions type once).
IMO need to be another PR if we want to allow multiple usage of rules.